### PR TITLE
Migrate to NcEmptyContent

### PR DESCRIPTION
**Pull request by ShGKme**
### ☑️ Resolves

Use `<NcEmptyContent>` instead our own copy.

### 🖼️ Screenshots / Screencasts

| 🏚️ Before | 🏡 After |
|------------|----------|
image | image
image | image
image | image

### 🏁 Checklist

- [x] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
- [x] 🖌️ Design was reviewed, approved or inspired by the design team
- [x] ⛑️ Tests are included or not possible
- [x] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required

### CODE REVIEW

**Review by Antreesy — APPROVED** [review]

File: src/views/SessionConflictView.vue [hunk 1]
```diff
@@ -1,17 +1,25 @@
 <template>
-	<div id="emptycontent">
-		<div id="emptycontent-icon" class="icon-info" />
-		<h2>{{ t('spreed', 'Duplicate session') }}</h2>
-		<p class="emptycontent-additional">
-			{{ t('spreed', 'You joined the conversation in another window or device. This is currently not supported by Nextcloud Talk so this session was closed.') }}
-		</p>
-	</div>
+	<EmptyView :name="t('spreed', 'Duplicate session')"
```
[the remark this answers is not in the document]
Comment (ShGKme) [reply src/views/SessionConflictView.vue:2]: I only replaced the custom layout with Nc component layout, no logic changes...

[the remark this answers is not in the document]
Comment (Antreesy) [reply src/views/SessionConflictView.vue:2]: When it's not related, will raise an issue

**Review by DorraJaouad — APPROVED** [review]
Looks good, thank you 😸 .

**Comment by ShGKme** [comment]
Fixed stylelint issues.

Added `?raw` imports to ignore in ESLint, because it is not supported by `eslint-plugin-import`

**Pull request merged**